locations/spiders/alabama_department_of_transportation_us.py

```diff
@@ -0,0 +1,29 @@
+from typing import Iterable
+
+from scrapy.http import Response
+
+from locations.categories import Categories
+from locations.items import Feature
+from locations.json_blob_spider import JSONBlobSpider
+
+
+class AlabamaDepartmentOfTransportationUSSpider(JSONBlobSpider):
+    name = "alabama_department_of_transportation_us"
+    item_attributes = {
+        "operator": "Alabama Department of Transportation",
+        "operator_wikidata": "Q872788",
+        "state": "AL",
+        "extras": Categories.SURVEILLANCE_CAMERA.value,
+    }
+    allowed_domains = ["api.algotraffic.com"]
+    start_urls = ["https://api.algotraffic.com/v3.0/Cameras"]
+
+    def post_process_item(self, item: Feature, response: Response, feature: dict) -> Iterable[Feature]:
+        item["name"] = " ".join(
+            [feature["location"]["displayRouteDesignator"], feature["location"]["displayCrossStreet"]]
+        )
+        item["lat"] = feature["location"]["latitude"]
+        item["lon"] = feature["location"]["longitude"]
+        item["extras"]["contact:webcam"] = ";".join(filter(None, [feature["hlsUrl"], feature["imageUrl"]]))
```

Comment:
We can do this later, but I'd really like for the value of `contact:webcam` to be a single URL, not `;`-concatenated URLs. We should come up with another key if the URLs lead to different images.

Reply:
Should we use https://wiki.openstreetmap.org/wiki/Multiple_values#Numbered_suffix_in_key or https://wiki.openstreetmap.org/wiki/Lanes#Two_driving_directions (two semicolons in the tag name) as an alternative to https://wiki.openstreetmap.org/wiki/Multiple_values#Semi-colon_value_separator ?

By comparison, `contact:phone` seems to use semicolons the most, almost never uses https://wiki.openstreetmap.org/wiki/Multiple_values#Numbered_suffix_in_key and occasionally uses https://wiki.openstreetmap.org/wiki/Lanes#Two_driving_directions for `contact:phone:US` vs. `contact:phone:GB` etc.